### DISCUSSION TITLE
Missing book spine

### DISCUSSION THREAD

**Discussion by sibguerra** [joaomlourenco/novathesis#310] [Q&A] [answered]
Hello, the book spine is missing, even when we compile the original template for PhD, available in FCT webpage. This happens also in Overleaf, so this is a problem that goes beyond my humble TeX editor...! How can we fix this problem? The book spine is mandatory: without it, the available template is uncomplete, and it it not accepted by NovaFCT... Thank you in advance.

**Comment by joaomlourenco** [comment, marked as answer]
Did you select "docstatus=final" in "Config/1_novathesis.tex"?

**Reply by sibguerra** [reply]
Done! But the subtitle does not appear in the book spine. I will confirm if the subtitle is mandatory, because the document that stablishes the setting of the book spine is not clear about it... Thank you again!

![lombada](https://user-images.githubusercontent.com/100953769/156811730-423f09e7-b140-4ab1-8d43-7fdf3372e63e.jpg)

**Reply by joaomlourenco** [reply]
You can't put the "Rossio" in the "Rua da Betesga"… which means: you can't put a VERY VRY VERY VERY long tile (and subtitle) in a limited space.

Suggestion: Leave it with just the title and submit.  Let people complain that something is missing.  I bet no one will complain.  ;)

**Reply by sibguerra** [reply]
Ok! I will do as you suggest! 😄
